diff --git a/Casks/i/iterm2.rb b/Casks/i/iterm2.rb
index d055d74fa9b..e2e3133e015 100644
--- a/Casks/i/iterm2.rb
+++ b/Casks/i/iterm2.rb
@@ -1,33 +1,14 @@
 cask "iterm2" do
-  # NOTE: "2" is not a version number, but an intrinsic part of the product name
-  on_high_sierra :or_older do
-    version "3.3.12"
-    sha256 "6811b520699e8331b5d80b5da1e370e0ed467e68bc56906f08ecfa986e318167"
+  version "3.5.4"
+  sha256 "acfb8f93550277c231c572a488cd0290e74bce538354a5f5515a889ea4b44bd5"
 
-    livecheck do
-      skip "Legacy version"
-    end
-  end
-  on_mojave do
-    version "3.4.16"
-    sha256 "b0941a008ead9f680e9f4937698b9b849acbb4e30ed1f3f100e3616cd6d49c0b"
-
-    livecheck do
-      skip "Legacy version"
-    end
-  end
-  on_catalina :or_newer do
-    version "3.5.4"
-    sha256 "acfb8f93550277c231c572a488cd0290e74bce538354a5f5515a889ea4b44bd5"
-
-    livecheck do
-      # workaround for
-      # - https://github.com/Homebrew/homebrew-cask/pull/104019
-      # - https://github.com/gnachman/iterm2-website/issues/82
-      # url "https://iterm2.com/appcasts/final_modern.xml"
-      url "https://raw.githubusercontent.com/gnachman/iterm2-website/master/source/appcasts/final_modern.xml"
-      strategy :sparkle
-    end
+  livecheck do
+    # workaround for
+    # - https://github.com/Homebrew/homebrew-cask/pull/104019
+    # - https://github.com/gnachman/iterm2-website/issues/82
+    # url "https://iterm2.com/appcasts/final_modern.xml"
+    url "https://raw.githubusercontent.com/gnachman/iterm2-website/master/source/appcasts/final_modern.xml"
+    strategy :sparkle
   end
 
   url "https://iterm2.com/downloads/stable/iTerm2-#{version.dots_to_underscores}.zip"
diff --git a/Casks/w/whatsapp.rb b/Casks/w/whatsapp.rb
index bd064af70e7..18775caf9ed 100644
--- a/Casks/w/whatsapp.rb
+++ b/Casks/w/whatsapp.rb
@@ -1,8 +1,8 @@
 cask "whatsapp" do
-  version "2.24.16.80"
-  sha256 "33026f9cc9b1c63010df546c1ac5ff3b49ff6e742a57807a817264db82487ce9"
+  version "2.25.36.33"
+  sha256 "9098d8790ac5126b14240850fbe504dd7c9075579304a35e3deeebf552a53d28"
 
-  url "https://web.whatsapp.com/desktop/mac_native/release/?version=#{version}&extension=zip&configuration=Release&branch=relbranch"
+  url "https://web.whatsapp.com/desktop/mac_native/release/?version=#{version}&extension=zip&configuration=Release&branch=master&is_buck=true"
   name "WhatsApp"
   desc "Native desktop client for WhatsApp"
   homepage "https://www.whatsapp.com/"
